### ISSUE TITLE
Smart Playlist "Recently Added" -- Does not return results no matter what value.  Generates SQL error in log file

### ISSUE THREAD

**Issue by tim-e-de-vries** [ampache/ampache#2585]
## Description
Smart Playlist "Recently Added" -- Does not return results no matter what value.  Generates SQL error in log file

### Describe the bug

Trying to figure out how to use "Recently added/updated" fields in the Smart Playlist.  No joy for getting any search results with any numeric value.  Checking the logs (/var/log/ampache/....) get the following SQL error:
 (dba.class) -> Error_query MSG: ["42000",1064,"You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '>= DESC LIMIT 7766) as `addition_time_89208035acaef86f343adafb20857790` ON `song' at line 1"] 

### To reproduce

1. Go to 'New Smart Playlist'
2. Click on 'any filter row'
3. Scroll down to 'Recently Added/Updated'
4. Click Search
5. Review Ampache log file for errors 

### Expected behavior

Would expect to see songs that match criteria (Songs that were added to the library for a given number of seconds greater than/lesser than now??)
Where would I find a definition of what the criteria mean?

## Environment

* Ampache version:  4.2.3-develop
* Web  Apache/2.4.41 (Ubuntu):
* Server operating system Ubuntu 20.04:
* Client operating system Chrome on Linux:

### Client type

* Web
* Ampache
## Logs
```
2020-10-19T21:06:07-04:00 [admin] (dba.class) -> Error_query SQL: SELECT DISTINCT(`song`.`id`) FROM `song` LEFT JOIN (SELECT `id` from `song` ORDER BY >= DESC LIMIT 7766) as `addition_time_89208035acaef86f343adafb20857790` ON `song`.`id` = `addition_time_89208035acaef86f343adafb20857790`.`id` WHERE `addition_time_89208035acaef86f343adafb20857790`.`id` IS NOT NULL
2020-10-19T21:06:07-04:00 [admin] (dba.class) -> Error_query MSG: ["42000",1064,"You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '>= DESC LIMIT 7766) as `addition_time_89208035acaef86f343adafb20857790` ON `song' at line 1"]
2020-10-19T21:06:07-04:00 [admin] (dba.class) -> Error_query SQL: SELECT DISTINCT(`song`.`id`) FROM `song` LEFT JOIN (SELECT `id` from `song` ORDER BY >= DESC LIMIT 7766) as `addition_time_89208035acaef86f343adafb20857790` ON `song`.`id` = `addition_time_89208035acaef86f343adafb20857790`.`id` WHERE `addition_time_89208035acaef86f343adafb20857790`.`id` IS NOT NULL
2020-10-19T21:06:07-04:00 [admin] (dba.class) -> Error_query MSG: ["42000",1064,"You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '>= DESC LIMIT 7766) as `addition_time_89208035acaef86f343adafb20857790` ON `song' at line 1"]
2020-10-19T21:06:07-04:00 [admin] (dba.class) -> Error_query SQL: SELECT DISTINCT(`song`.`id`) FROM `song` LEFT JOIN (SELECT `id` from `song` ORDER BY >= DESC LIMIT 7766) as `addition_time_89208035acaef86f343adafb20857790` ON `song`.`id` = `addition_time_89208035acaef86f343adafb20857790`.`id` WHERE `addition_time_89208035acaef86f343adafb20857790`.`id` IS NOT NULL
2020-10-19T21:06:07-04:00 [admin] (dba.class) -> Error_query MSG: ["42000",1064,"You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '>= DESC LIMIT 7766) as `addition_time_89208035acaef86f343adafb20857790` ON `song' at line 1"]
2020-10-19T21:06:07-04:00 [admin] (dba.class) -> Error_query SQL: SELECT DISTINCT(`song`.`id`) FROM `song` LEFT JOIN (SELECT `id` from `song` ORDER BY >= DESC LIMIT 7766) as `addition_time_89208035acaef86f343adafb20857790` ON `song`.`id` = `addition_time_89208035acaef86f343adafb20857790`.`id` WHERE `addition_time_89208035acaef86f343adafb20857790`.`id` IS NOT NULL
2020-10-19T21:06:07-04:00 [admin] (dba.class) -> Error_query MSG: ["42000",1064,"You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '>= DESC LIMIT 7766) as `addition_time_89208035acaef86f343adafb20857790` ON `song' at line 1"]
2
```

**Comment by lachlan-00** [comment]
it's the wrong type, it shouldn't be asking for more than the total
![image](https://user-images.githubusercontent.com/1305249/96541129-aed47480-12e2-11eb-94f0-3c832dc8629e.png)

**Comment by lachlan-00** [comment]
fixed in 9ee218480
